ViewLayer/CompactView/CompactView.cpp

```diff
@@ -0,0 +1,826 @@
+#include "CompactView.h"
+#include "../../PresenterLayer/BatteryPresenter/BatteryPresenter.h"
+#include "../../PresenterLayer/CommunicationPresenter/CommunicationPresenter.h"
+#include "../../PresenterLayer/PowerPresenter/PowerPresenter.h"
+#include "../../PresenterLayer/VehiclePresenter/VehiclePresenter.h"
+#include "../../PresenterLayer/MpptPresenter/MpptPresenter.h"
+#include "../../PresenterLayer/FaultsPresenter/FaultsPresenter.h"
```

Comment:
For future reference, you can just have `#include "PresenterLayer/FaultsPresenter/FaultsPresenter.h"` the file paths can just be from the include directory and not relative from the file.